CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+
+* Updated development dependencies.
+
+**olp-sdk-dataservice-read**
+
+* Added method for fetching partitions metadata for specific partition IDs.
```

Comment:
Please specify which method it is, e.g. CPP SDK:

"Added new class VersionedLayerClient that is used to access versioned layers in OLP. Class implements GetData, GetPartitions, PrefetchTiles methods from CatalogClient."

Reply:
updated